examples/Serving-An-XGboost-Model-With-Merlin-Systems.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>It the text <code>We do so in a subprocess in order to continue our exploration in this notebook.</code> still correct? I dont think we do this anymore? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/systems/pull/157/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Yes, absolutely! Not sure how I missed that, apologies. Removing it now